core/rootcling_stage1/src/rootcling_stage1.cxx

```diff
@@ -24,12 +24,22 @@ static void (*dlsymaddr)() = &usedToIdentifyRootClingByDlSym;
 ROOT::Internal::RootCling::TROOTSYSSetter gROOTSYSSetter;
 
 static const char *GetIncludeDir() {
-   static std::string incdir = std::string(getenv("ROOTSYS")) + "/include";
+   const char *rootsys = getenv("ROOTSYS");
```

Comment:
Should we instead assert that `ROOTSYS` is set at this point?

Reply:
Yes, I agree that it is a better idea since the variable should always be defined. As requested, the assertion is added together with a comment pointing to the place of definition.